.github/workflows/scripts/code_coverage_disable_check.py

```
@@ -0,0 +1,161 @@
+    )
+    try:
+        with open(file_path, "r", encoding="utf-8") as file:
+            content = file.read()
+            return bool(code_coverage_disable_pattern.search(content))
+    except FileNotFoundError:
+        print(f"File not found: {file_path}")
+        return False
+    except PermissionError:
+        print(f"Permission denied: {file_path}")
+        return False
+    except (IOError, OSError) as e:
```

Ruff (0.8.2)

Replace aliased errors with `OSError`

Replace with builtin `OSError`

(UP024)